profiling/src/internal/observation/observations.rs

```diff
@@ -68,19 +83,31 @@ impl Observations {
                 .aggregated_data
                 .iter()
                 .map(move |(sample, obs)| (sample, None, obs));
-            let timestamped_data = observations
-                .timestamped_data
-                .iter()
-                .map(move |(sample, ts, obs)| (sample, Some(*ts), obs));
+            // let timestamped_data = observations
```

Comment:
Lets talk about how to make this work again :)

Reply:
Fixed it!